runtime/reconcilers/metrics_view.go

```diff
@@ -152,6 +153,27 @@ func (r *MetricsViewReconciler) validate(ctx context.Context, mv *runtimev1.Metr
 	return errors.Join(errs...)
 }
 
+func validateDimension(ctx context.Context, olap drivers.OLAPStore, t *drivers.Table, d *runtimev1.MetricsViewSpec_DimensionV2, fields map[string]*runtimev1.StructType_Field) error {
+	if d.Expression == "" {
+		if d.Column == "" {
+			return nil
```

Comment:
Shouldn't this be an error? (And ideally checked also at parser level)

Reply:
True. I guess we will never hit this without the parser running and dont need backwards compatibility right?